x-pack/platform/test/api_integration/services/ml/test_resources_data.ts

```diff
@@ -5,55 +5,71 @@
  * 2.0.
  */
 
-export const savedSearches = {
+import type { DiscoverSessionAttributes } from '@kbn/saved-search-plugin/server/saved_objects/schema';
```

Comment:
```suggestion
import type { DiscoverSessionAttributes } from '@kbn/saved-search-plugin/server';
```
Nit: Since we're using this type outside of the plugin, might make sense to export it from the root.